dependencies/src/main/java/com/google/cloud/tools/opensource/classpath/ClassFile.java

```diff
@@ -23,8 +23,8 @@
 import java.util.Objects;
 
 /**
- * A tuple of a class name and a jar file to uniquely locate the class implementation in a class
- * path.
+ * A compiled class file of {@code className} in {@code jar} to uniquely locate the class
```

Comment:
Perahps, "a locator for a compiled class file in a jar"

Reply:
Nice. Updated.